src/mscorlib/src/System/AppDomainSetup.cs

```diff
@@ -196,11 +185,7 @@ public String TargetFrameworkName
         {
             get
             {
-                return _TargetFrameworkName;
-            }
-            set
-            {
-                _TargetFrameworkName = value;
+                return null;
```

Comment:
Interesting. I will break out an issue since this will cause AppContext to parse ".NETCoreApp,Version=v2.0" and have an identifier and version instead of empty/null as it currently has.

Accordingly I will leave this return null here since the easiest fix would paste `                return Assembly.GetEntryAssembly()?.GetCustomAttribute<TargetFrameworkAttribute>()?.FrameworkName;` into AppDomainSetup here - that's where AppContextDefaultValues gets it form.